### DISCUSSION TITLE
how to backup to a remote FTP server

### DISCUSSION THREAD

**Discussion by grinch-tcrc** [spatie/laravel-backup#1513] [General]
Hi, I am trying to store my backup file in multiple destinations, first one is my local system and the second is a remote server. The backup file is successfully stored to my local system but I am getting connection error storing backup file to the remote server. Basically spatie/laravel-backup is unable to connect to the remote ftp server even though the server credential and details are all correct.  Can someone please help me understand what is going wrong here and how to fix it.

**ERROR Message:**
`Copying zip failed because: Could not connect to host: BACKUP_FTP_HOST, port:FTP_PORT.`

**using spatie/laravelbackup version 7.8.0.**

**config/filesystem.php** 

```
 'ftp' => [
            'driver'   => 'ftp',
            'host'     => env('BACKUP_FTP_HOST'),
            'username' => env('BACKUP_FTP_USERNAME'),
            'password' => env('BACKUP_FTP_PASSWORD'),
            'root' => '/home/grinch/Desktop/backup', 
            // Optional FTP Settings...
            'port'     => FTP_PORT,
            // 'root'     => '',
            // 'passive'  => true,
            // 'ssl'      => true,
            // 'timeout'  => 30,
        ],
```

**and included the diver in config/backup.php**

```
'destination' => [
    'disks' => [
                    // 'local',
                    'local2',
                    'ftp',
                ],
],
```

**Comment by erikn69** [comment]
Why `'port'     => FTP_PORT,`, not must be `'port' => env('FTP_PORT', 21),`?

**Reply by Mdhesari** [reply]
it's optional as said in comment
